Frequently_used_code/ARD_overpass_predictor.ipynb

```diff
@@ -0,0 +1,1677 @@
+{
```

Comment:
<p>The Landsat, Sentinel 2A, and Sentinel 2B code are all very similar. Can you use a dictionary and make this into a loop, so that the code isn't repeated? Alternatively you could write a function that operated on one satellite, and then apply it for all three.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GeoscienceAustralia/dea-notebooks/pull/736/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>